LogosLinuxInstaller.py

```diff
@@ -102,153 +175,124 @@ def parse_args(args):
         if os.path.isdir(args.custom_binary_path):
             config.CUSTOMBINPATH = args.custom_binary_path
         else:
-            cli_msg(f"{config.LOGOS_SCRIPT_TITLE}: User supplied path: \"{args.custom_binary_path}\". Custom binary path does not exist.")
-            parser.print_help()
-            sys.exit(1)
+            parser.exit(status=1, message=f"Custom binary path does not exist: \"{args.custom_binary_path}\"\n")
 
-    if args.indexing:
-        config.ACTION = 'indexing'
     if args.passive:
         config.PASSIVE = True
-    if args.logs:
-        config.ACTION = 'logging'
-    if args.control_panel:
-        config.ACTION = 'control'
-
-def run_control_panel():
-    classname = "LogosLinuxControlPanel"
-    app = App(className=classname)
-    win = ControlWindow(app, class_=classname)
-    app.mainloop()
 
-def edit_config():
-    open_config_file()
+    # Set ACTION function.
+    if args.install_app:
+        config.ACTION = installer.install
+    elif args.run_installed_app:
+        config.ACTION = run_logos
+    elif args.run_indexing:
+        wine.run_indexing
+    elif args.remove_library_catalog:
+        control.remove_library_catalog
+    elif args.remove_index_files:
+        control.remove_all_index_files
+    elif args.edit_config:
+        config.ACTION = control.edit_config
+    elif args.reinstall_dependencies:
+        config.ACTION = utils.check_dependencies
+    elif args.backup:
+        config.ACTION = control.backup
+    elif args.restore:
+        config.ACTION = control.restore
+    elif args.set_appimage:
+        utils.set_appimage
+    elif args.get_winetricks:
+        config.ACTION = control.get_winetricks
+    elif args.run_winetricks:
+        config.ACTION = wine.run_winetricks
+    elif args.toggle_app_logging:
+        config.ACTION = wine.switch_logging
+    elif args.create_shortcuts:
+        config.ACTION = installer.create_shortcuts
+    elif args.remove_install_dir:
+        config.ACTION = control.remove_install_dir
+    else: # default function if app is installed
+        config.ACTION = run_control_panel
```

Comment:
Do we need to do checks that verify these things are installed? As in, error if no Logos install found?